MoltenVK/MoltenVK/GPUObjects/MVKInstance.mm

```diff
@@ -334,7 +334,13 @@
 	initDebugCallbacks(pCreateInfo);	// Do before any creation activities
 
 	mvkSetOrClear(&_appInfo, pCreateInfo->pApplicationInfo);
-	if (_appInfo.apiVersion == 0) { _appInfo.apiVersion = VK_API_VERSION_1_0; }	// Default
+    
+    if (_appInfo.apiVersion > MVK_VULKAN_API_VERSION) {
```

Comment:
The spec dictates we should ignore patch version. An app coming in with a higher patch version would fail this test inappropriately. Should we apply `MVK_VULKAN_API_VERSION_CONFORM()` to both sides of this test? 

And the warning message indicates we're going to assume MoltenVK's level. Should we not be correcting `_appInfo` to that value, in this case?

Reply:
> Should we apply MVK_VULKAN_API_VERSION_CONFORM() to both sides of this test?

Yes, that would be more correct.

> Should we not be correcting _appInfo to that value, in this case?

In my opinion, if we are already correcting the smaller value, then the bigger should also be corrected.

Reply:
>In my opinion, if we are already correcting the zero value, then the bigger should also be corrected.

Agreed. Are you able to fix that here?

Reply:
Of course! :) fae1bb53